inapppy/googleplay.py

```diff
@@ -122,21 +128,31 @@ def _ms_timestamp_expired(ms_timestamp: str) -> bool:
 
         return datetime.datetime.utcfromtimestamp(ms_timestamp_value) < now
 
+    @staticmethod
+    def _create_credentials(private_key: Union[str, dict], scope_str: str):
+        # If str, assume its a filepath
+        if isinstance(private_key, str):
+            if not os.path.exists(private_key):
+                raise InAppPyError(f"Google API private key file does not exist: {private_key}")
+            return ServiceAccountCredentials.from_json_keyfile_name(private_key, scope_str)
+        # If dict, assume parsed json
+        if isinstance(private_key, dict):
+            return ServiceAccountCredentials.from_json_keyfile_dict(private_key, scope_str)
+        raise InAppPyError(f"Unknown private key format: {repr(private_key)}, expected 'dict' or 'str' types")
```

Comment:
```suggestion
        raise InAppPyError(f"Unknown service account credential format: {repr(private_key)}, expected 'dict' or 'str' types")
```

Reply:
No problem.
However the variable is called 'private_key'

Reply:
Indeed - what would you think of renaming the variable to something like `credentials`, since the file/dict contains more than just the private key.

This way the public API and the implementation would be consistent.